docs/resources/images_image_copy.md

```diff
@@ -0,0 +1,73 @@
+---
+subcategory: "Image Management Service (IMS)"
+---
+
+# huaweicloud_images_image_copy
+
+Manages an IMS image copy resources within HuaweiCloud.
+
+## Example Usage
```

Comment:
it's great to add two examples: with-in region and cross region

Reply:
done